src/ripple/app/misc/impl/ValidatorList.cpp

```diff
@@ -395,6 +395,9 @@ ValidatorList::removePublisherList (PublicKey const& publisherKey)
             --iVal->second;
     }
 
+    iList->second.list.clear();
```

Comment:
Without this call to `clear`, would that mean if a published list with keys `K1,K2` expired, then was republished with key `K1`, the `keyListsings_` count for `K2` would have been decremented twice? If so, is it worth adding a unit test to cover that case?

Reply:
Good observation. This added test fails without the `clear`
https://github.com/ripple/rippled/pull/2240/commits/3bb20f990330afe60e043a53acacccc939c0a222